xml/System.Security.Cryptography/DSA.xml

```diff
@@ -727,10 +727,20 @@ The algorithm-specific key import failed.
         <Parameter Name="passwordBytes" Type="System.ReadOnlySpan&lt;System.Byte&gt;" Index="1" FrameworkAlternate="net-5.0" />
       </Parameters>
       <Docs>
-        <param name="input">To be added.</param>
-        <param name="passwordBytes">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <param name="input">The PEM text of the encrypted key to import.</param>
+        <param name="passwordBytes">The bytes to use as a password when decrypting the key material.</param>
+        <summary>Imports an encrypted RFC 7468 PEM-encoded private key, replacing the keys for this object.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+ The password bytes are passed directly into the Key Derivation Function (KDF) used by the algorithm indicated by <c>pbeParameters</c>. This enables compatibility with other systems which use a text encoding other than UTF-8 when processing passwords with PBKDF2 (Password-Based Key Derivation Function 2).
+  Unsupported or malformed PEM-encoded objects will be ignored. If multiple supported PEM labels are found, an exception is thrown to prevent importing a key when the key is ambiguous.
```

Comment:
These remarks should be separate paragraphs, which (IIRC) requires a blank line between them:


```C#
        /// <remarks>
        ///   <para>
        ///   The password bytes are passed directly into the Key Derivation Function (KDF)
        ///   used by the algorithm indicated by <c>pbeParameters</c>. This enables compatibility
        ///   with other systems which use a text encoding other than UTF-8 when processing
        ///   passwords with PBKDF2 (Password-Based Key Derivation Function 2).
        ///   </para>
        ///   <para>
        ///   Unsupported or malformed PEM-encoded objects will be ignored. If multiple supported PEM labels
        ///   are found, an exception is thrown to prevent importing a key when
        ///   the key is ambiguous.
        ///   </para>
        ///   <para>This method supports the <c>ENCRYPTED PRIVATE KEY</c> PEM label.</para>
        /// </remarks>
```

=>

```md
        The password bytes are passed directly into the Key Derivation Function (KDF)
        used by the algorithm indicated by <c>pbeParameters</c>. This enables compatibility
        with other systems which use a text encoding other than UTF-8 when processing
        passwords with PBKDF2 (Password-Based Key Derivation Function 2).

        Unsupported or malformed PEM-encoded objects will be ignored. If multiple supported PEM labels
        are found, an exception is thrown to prevent importing a key when
        the key is ambiguous.
        
        This method supports the <c>ENCRYPTED PRIVATE KEY</c> PEM label.
```

Reply:
Thanks. The tool should avoid removing `<para></para>` from triple slash comments.

Reply:
I fixed the tool so that `<para>` tags are substituted with a double endline.